docs/ldml/tr35-keyboards.md

```diff
@@ -1387,9 +1400,15 @@ Layers are matched exactly based on the modifier keys which are down. For exampl
 
 - `other` as a modifier will match if no other layers match.
 
-Multiple modifier sets may be separated by commas.  For example, `none, shift caps` will match either no modifiers *or* shift and caps.  `ctrlL altL, altR` will match either  left-control and left-alt, *or* right-alt.
+Multiple modifier sets are separated by commas.  For example, `none, shift caps` will match either no modifiers *or* shift and caps.  `ctrlL altL, altR` will match either  left-control and left-alt, *or* right-alt.
+
+Keystrokes processed where there isn’t an explicitly matching layer, and where there is no layer with `other` specified, are ignored.
```

Comment:
What does "keystrokes" mean here? Do you ignore the key pressed if it does not result in a layer? For example, if there is a layer `ctrl` and you press `ctrl` and then add `shift`, the shift is ignored and layer `ctrl` matches? Perhaps it's just wording, "key combinations"? It would be good to add some example. "For example, if only `none`, `caps` and `shift` layers exist, pressing `caps` and `shift` will result in no output.

Reply:
Yes, it's then ignored if it doesn't result in a layer. I'll add an example.